app/src/app/views/stix/notes-search/notes-search-list/notes-search-list.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import { SidebarService } from 'src/app/services/sidebar/sidebar.service';
+
+@Component({
+  selector: 'app-notes-search-list',
+  templateUrl: './notes-search-list.component.html',
+  styleUrls: ['./notes-search-list.component.scss']
+})
+export class NotesSearchListComponent implements OnInit {
+  constructor(private sidebarService: SidebarService) { 
+  }
+
+  ngOnInit(): void {
+    this.sidebarService.opened = false;
```

Comment:
Closing the sidebar here causes an `ExpressionChangedAfterItHasBeenChecked` error to be thrown when the `notes-search` page loads from a previous page where the resource drawer is already opened. The resource drawer should remain open if it's been opened by the user.

![image](https://user-images.githubusercontent.com/48832936/225704447-451c2d4b-9f16-45b5-a0e2-45d87b47f909.png)